src/ProxyClient/Varnish.php

```diff
@@ -109,7 +109,7 @@ public function invalidateTags(array $tags)
      */
     public function getTagsHeaderValue(array $tags)
     {
-        return implode(',', array_unique($this->escapeTags($tags)));
+        return array_unique($this->escapeTags($tags));
```

Comment:
The difference is that we don't join the tags into one header ourselves, as PSR implementations already be do that (see ResponseInterface).